src/test/INTEGRATION_TEST.txt

```diff
@@ -0,0 +1,41 @@
+﻿Product: Integration tests for WSO2 ESB cerner connector through REST API
+
+Pre-requisites:
+	-Maven 3.x
+	-Java 1.8
+
+Tested Platform:
+	- ubuntu 16.04
+	- WSO2 EI 6.3.0
+
+	Steps to follow in setting integration test.
+	
+	1. Download WSO2 EI 6.3.0 from official website.
+
+	2. Enable the following message formatter and message builder in the WSO2 EI axis2.xml
+		<messageFormatter contentType="application/json+fhir" class="org.apache.synapse.commons.json.JsonStreamFormatter"/>
+		<messageBuilder contentType="application/json+fhir" class="org.apache.synapse.commons.json.JsonStreamBuilder"/>
+
+	3. Extract the certificate from browser by navigating to the official cerner sandbox URL and Add the certificate obtained to "<EI_HOME>/repository/resources/security" folder and "{CERNER_HOME}/src/test/resources/keystores/products".
+
+	i)"<CERNER_HOME>/src/test/resources/keystores/products"
+
+	        Navigate to the above location from command prompt and execute
+            'keytool -importcert -file CERT_FILE_NAME -keystore wso2carbon.jks -alias "cernercert"' in command line to import cerner certificate in to keystore. Give "wso2carbon" as password.
+            NOTE : CERT_FILE_NAME is the file name which was extracted from cerner with  the extension, change it accordingly.
+            Remove the copied certificate.
+
+	ii) "WSO2-EI-6.3.0/repository/resources/security"
+
+           	Navigate to the above location from command prompt and execute 'keytool -importcert -file CERT_FILE_NAME -keystore client-truststore.jks -alias "cernercert"' in command line to 				import cerner certificate in to keystore. Give "wso2carbon" as password.
+            NOTE : CERT_FILE_NAME is the file name which was extracted from cerner with the extension, change it accordingly. Remove the copied certificate.
+			CERT_NAME is arbitrary name for the certificate.
```

Comment:
If you upgrade the integration-base to 1.0.3 version, you don't need this steps to add the certificate to EI. It is enough to put the certificate to connector_home/repository and update the pom.xml. please refer this PR for detail [1].

[1] https://github.com/wso2-extensions/esb-connector-salesforce/pull/25/files#diff-42378125e2f0963d4ab4c358409b7529R46